docs/reactivity/key-concepts.md

```diff
@@ -0,0 +1,76 @@
+# Key Concepts of Reactivity
+
+## Working with reactive data streams
+
+Reactive data streams are a fundamental concept in Oryx, and play a crucial role in managing and manipulating data in real-time. Oryx prefers Observables over Promises as they are more powerful and allow for continuous streams of data over time. This is particular helpful in experiences that remain active during some time, for example in a Single Page Application (SPA).
+
+An observable can emit different values over time. In Oryx, components are bind to data observed from APIs and stored in the [application state](#application-state). Whenever the application state is updated, a new value is emitted and the component will update its view automatically in an efficient manner.
+
+Oryx makes use of reactive programming through the popular library [RxJS](https://rxjs.dev/). It provides a set of tools and techniques to make it easier to work with asynchronous data streams and event-driven systems. RxJS is an platform agnostic library that provides:
+
+- an Observable primitive that can be used as a base mechanism for reactivity with support or bridges into - different frameworks (vue, react, Lit, svelte, etc.)
+- ability to emit more than one value (unline Promises that are resolved only once)
+- ability to resolve value both asynchronously and synchronously (Promises can resole only asynchronously)
```

Comment:
We already explained that Observables are better than Promises. Is there any reason to keep mentioning promises?